src/network/networkd-address.h

```diff
@@ -53,6 +53,11 @@ struct Address {
         union in_addr_union in_addr_peer;
 
         bool ip_masquerade_done:1;
+        bool home;
+        bool nodad;
+        bool mngtmpaddr;
+        bool noprefixroute;
+        bool autojoin;
```

Comment:
`ip_masquerade_done` is not parsed from the config files hence that will work. But in case of other I don't think we have something like `config_parse_bit()`. I think that will eventually overwrite the next variable.